code/Modules/Roles/Faction.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace ssl.Modules.Roles
+{
+    /// <summary>
+    /// Factions of the game, the main scenario is Protagonists vs Traitors.
+    /// The game stops if a faction wins and a role can have multiple factions.
+    /// </summary>
+    public enum Faction
+    {
+        /// <summary>
+        /// Good guys who maintain the station, the main faction
+        /// </summary>
+        Protagonists, 
+        /// <summary>
+        /// Want to distrub the station
+        /// </summary>
+        Traitors,
+        /// <summary>
+        /// Want to kill everyone
+        /// </summary>
+        Killer, 
+    }
```

Comment:
suggestion(non-blocking): We shouldn't hardcode the factions, i'm fine with it for now but I let you create the ticket if you think we'll do it later